binaryVersion

```diff
@@ -1 +1 @@
-tags/v1.3.17
+feat/release-2022-April
```

Comment:
no need for a normal binaryVersion here?

Reply:
I think we should use a tag

Reply:
right, put tags/v1.3.19
Thanks